source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -166,23 +179,31 @@ class FilterConfig {
 
   const Envoy::ProtobufWkt::Struct& filterMetadata() const { return filter_metadata_; }
 
+  const absl::optional<ProtobufWkt::Struct>
+  evaluateRequestAttributes(const Filters::Common::Expr::ActivationPtr& activation) const {
+    return expression_manager_.evaluateRequestAttributes(std::move(activation));
+  }
+
+  const absl::optional<ProtobufWkt::Struct>
+  evaluateResponseAttributes(const Filters::Common::Expr::ActivationPtr& activation) const {
+    return expression_manager_.evaluateResponseAttributes(std::move(activation));
+  }
+
+  bool hasRequestExpr() const { return expression_manager_.hasRequestExpr(); }
+
+  bool hasResponseExpr() const { return expression_manager_.hasResponseExpr(); }
```

Comment:
should probably just expose the expression manager at this point
```suggestion
  const ExpressionManager& expressionManager() { return expression_manager_; }
```